### PR TITLE
Search request price

### PR DESCRIPTION
**Pull request by muljo**
Not sure what type the maxPrice and minPrice should be.  
The client side on the AllGoods website converts float values to an integer, but the server side API accepts integers.

### CODE REVIEW

**Comment by muljo** [comment]
Will create a real PR after I get feedback.

File: app/src/main/java/com/sthoray/allright/data/model/search/SearchRequest.kt [hunk 1]
```diff
@@ -36,9 +36,11 @@ data class SearchRequest(
     @SerializedName("free_shipping")
     var freeShipping: Int = 0,
     var location: Any = Any(),
-    var max_price: Float? = null,   // Client side casts max price and min price to an int
+    @SerializedName("max_price")
+    var maxPrice: Float? = null,   // Client side casts max price and min price to an int
                              // but the server side accepts float values
```
Comment (scsole) [inline app/src/main/java/com/sthoray/allright/data/model/search/SearchRequest.kt]: Let's leave them as `Floats`. Could the comments be moved into the property kdoc?

Comment (muljo) [reply app/src/main/java/com/sthoray/allright/data/model/search/SearchRequest.kt]: @scsole will do

File: app/src/main/java/com/sthoray/allright/data/model/search/SearchRequest.kt [hunk 2]
```diff
@@ -16,6 +16,10 @@ import com.sthoray.allright.utils.SortOrder
  * @property fastShipping 1 if fast shipping is required, else 0
  * @property freeShipping 1 if free shipping is required, else 0
  * @property location the location to search around
+ * @property maxPrice the maximum price of the listings
+ * Client side casts max price to an int, but the server side API accepts float values
```
Comment (scsole) [inline app/src/main/java/com/sthoray/allright/data/model/search/SearchRequest.kt:20]: This generates the following output: 

```
maxPrice - the maximum price of the listings Client side casts max price to an int, but the server side API accepts float values
```

See if multiline comments can be done or add some punctuation

Comment (muljo) [reply app/src/main/java/com/sthoray/allright/data/model/search/SearchRequest.kt:20]: With HTML tags?

**Pull request merged**